start.php

```diff
@@ -31,11 +31,36 @@
 
 	global $fs_active_plugins;
 
-	$this_sdk_relative_path = plugin_basename( dirname( __FILE__ ) );
+    $paths = array(

```

Comment:
The paths are in the `$paths` variable to avoid duplicating the `if-else` block that normalizes them. The logic is related to the way the `fs-essential-functions.php` file is included. If I put the normalization code after line 63 as you mentioned, the paths will not be normalized in the theme's SDK's starter file if a plugin that uses an older SDK has already set the `$fs_active_plugins` global variable.

It seems that I have to deal with or check the way the `fs-essential-functions.php` file is included first so that I can modify the logic related to the `$paths` variable.